packages/unmock-node/src/__tests__/loaders/fs-service-def-loader.test.ts

```diff
@@ -0,0 +1,52 @@
+import path from "path";
+import { IServiceDef } from "unmock-core";
+import { FsServiceDefLoader } from "../../loaders/fs-service-def-loader";
+
+const RESOURCES_DIR = path.join(__dirname, "resources");
+
+describe("File system service def loader", () => {
+  it("loads serviceDefs from existing directory", () => {
+    const serviceDefLoader = new FsServiceDefLoader({
+      servicesDir: RESOURCES_DIR,
+    });
+    const serviceDefs: IServiceDef[] = serviceDefLoader.loadSync();
+    expect(serviceDefs).toHaveLength(1);
+    const serviceDef = serviceDefs[0];
+    expect(serviceDef.directoryName).toBe("petstore");
+  });
+  it("loads serviceDefs from existing directory asynchronously", async () => {
+    const serviceDefLoader = new FsServiceDefLoader({
+      servicesDir: RESOURCES_DIR,
+    });
+    const serviceDefs: IServiceDef[] = await serviceDefLoader.load();
+    expect(serviceDefs).toHaveLength(1);
+    const serviceDef = serviceDefs[0];
+    expect(serviceDef.directoryName).toBe("petstore");
+  });
+
+  it("throws for a non-existing directory", () => {
+    const serviceDefLoader = new FsServiceDefLoader({
+      servicesDir: "DEFINITELY_DOES_NOT_EXIST_I_HOPE",
+    });
+    try {
+      serviceDefLoader.loadSync();
+    } catch (e) {
+      expect(/does not exist/.test(e.message)).toBe(true);
+      return;
```

Comment:
`expect(serviceDefLoader.loadSync).toThrow("does not exist");`

Reply:
True, I only had the async version earlier that worked better with the try-catch 👍